src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/RankSumUnitTest.java

```diff
@@ -0,0 +1,116 @@
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public final class RankSumUnitTest {
+
+    List<Integer> distribution20, distribution30, distribution20_40;
+    static final int observations = 100;
+
+    @BeforeClass
+    public void init() {
+        distribution20 = new ArrayList<>(observations);
+        distribution30 = new ArrayList<>(observations);
+        distribution20_40 = new ArrayList<>(observations);
+
+        final int skew = 3;
+        makeDistribution(distribution20, 20, skew, observations);
+        makeDistribution(distribution30, 30, skew, observations);
+        makeDistribution(distribution20_40, 20, skew, observations/2);
+        makeDistribution(distribution20_40, 40, skew, observations/2);
+
+        // shuffle the observations
+        Collections.shuffle(distribution20, Utils.getRandomGenerator());
+        Collections.shuffle(distribution30, Utils.getRandomGenerator());
+        Collections.shuffle(distribution20_40, Utils.getRandomGenerator());
+    }
+
+    private static void makeDistribution(final List<Integer> result, final int target, final int skew, final int numObservations) {
+        final int rangeStart = target - skew;
+        final int rangeEnd = target + skew;
+
+        int current = rangeStart;
+        for ( int i = 0; i < numObservations; i++ ) {
+            result.add(current++);
+            if ( current > rangeEnd )
+                current = rangeStart;
+        }
+    }
+
+    @DataProvider(name = "DistributionData")
+    public Object[][] makeDistributionData() {
+        List<Object[]> tests = new ArrayList<>();
+
+        for ( final int numToReduce : Arrays.asList(0, 10, 50, 100) ) {
+            tests.add(new Object[]{distribution20, distribution20, numToReduce, true, "20-20"});
+            tests.add(new Object[]{distribution30, distribution30, numToReduce, true, "30-30"});
+            tests.add(new Object[]{distribution20_40, distribution20_40, numToReduce, true, "20/40-20/40"});
+
+            tests.add(new Object[]{distribution20, distribution30, numToReduce, false, "20-30"});
+            tests.add(new Object[]{distribution30, distribution20, numToReduce, false, "30-20"});
+
+            tests.add(new Object[]{distribution20, distribution20_40, numToReduce, false, "20-20/40"});
+            tests.add(new Object[]{distribution30, distribution20_40, numToReduce, true, "30-20/40"});
+        }
+
+        return tests.toArray(new Object[][]{});
+    }
+
+    @Test(enabled = true, dataProvider = "DistributionData")
+    public void testDistribution(final List<Integer> distribution1, final List<Integer> distribution2, final int numToReduceIn2, final boolean distributionsShouldBeEqual, final String debugString) {
+        final MannWhitneyUTest mwu = new MannWhitneyUTest(NaNStrategy.FIXED, TiesStrategy.RANDOM);
```

Comment:
Actually, I'm going to delete this class. It tests a 3rd party library code, which we dont' need to do. FYI: in GATK3 it tests an otherwise dead method `runTwoSidedTest` on `MannWhitneyU` and could be deleted too.